Source/JavaScriptCore/wasm/WasmCalleeGroup.h

```diff
@@ -202,7 +202,7 @@ class CalleeGroup final : public ThreadSafeRefCounted<CalleeGroup> {
 #endif
     RefPtr<IPIntCallees> m_ipintCallees;
     RefPtr<LLIntCallees> m_llintCallees;
-    HashMap<uint32_t, RefPtr<JSEntrypointCallee>, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>> m_jsEntrypointCallees;
+    HashMap<uint32_t, RefPtr<JITCallee>, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>> m_jsEntrypointCallees;
```

Comment:
Hmm, so we have a `!ENABLE(JIT)` stub for JSEntrypointCallee but not JITCallee, so this breaks the `!ENABLE(JIT) && ENABLE(WEBASSEMBLY)` build. `m_jsEntrypointCallees` itself is used outside of JIT paths, so does this mean that JITCallee should no longer be JIT-exclusive either?